Framework/src/TrendingTask.cxx

```diff
@@ -257,7 +257,7 @@ void TrendingTask::generatePlots()
     }
     auto c = drawPlot(plotConfig);
     mPlots[plotConfig.name].reset(c);
-    getObjectsManager()->startPublishing(c, PublicationPolicy::Once);
+    getObjectsManager()->startPublishing<true>(c, PublicationPolicy::Once);
```

Comment:
you should not need `<true>` in TrendingTask and SliceTrendingTask anymore, after you disabled the check for postprocessing tasks.